### PR TITLE
[WIP] Retry the assets package generation if a CDN fails

### PR DESCRIPTION
**Pull request by ilausuch**
We observed this kind of errors in the openqa test at install_from_git

```
[warn] [AssetPack] Unable to download https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css: Connect timeout
```

This PR introduces a previous step before start the server
where the asset package is generated with a retry

https://progress.opensuse.org/issues/95995

### CODE REVIEW

**Comment by ilausuch** [comment]
Tested locally forcing a problem 
```
tools/generate-packed-assets || (sleep 1; tools/generate-packed-assets || exit 1) || (sleep 1; tools/generate-packed-assets || exit 1)
[2021-07-28 10:41:36.40841] [47325] [warn] [AssetPack] Unable to download https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2: Not Found
Could not find input asset "https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2". at /usr/lib/perl5/vendor_perl/5.32.1/Mojolicious/Plugin/AssetPack.pm line 172.
[2021-07-28 10:41:37.69231] [47333] [warn] [AssetPack] Unable to download https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2: Not Found
Could not find input asset "https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2". at /usr/lib/perl5/vendor_perl/5.32.1/Mojolicious/Plugin/AssetPack.pm line 172.
[2021-07-28 10:41:38.95224] [47340] [warn] [AssetPack] Unable to download https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2: Not Found
Could not find input asset "https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css2". at /usr/lib/perl5/vendor_perl/5.32.1/Mojolicious/Plugin/AssetPack.pm line 172.
```

**Comment by ilausuch** [comment]
Testing on openQA

File: tests/install/openqa_webui.pm [hunk 1]
```diff
@@ -56,6 +57,7 @@ git clone https://github.com/os-autoinst/openQA.git
 cd openQA
 for p in $(cpanfile-dump); do echo -n "perl($p) "; done | xargs zypper --non-interactive in -C
 cpanm -nq --installdeps .
+tools/generate-packed-assets || (sleep 1; tools/generate-packed-assets || exit 1) || (sleep 1; tools/generate-packed-assets || exit 1)
```
Comment (okurz) [inline tests/install/openqa_webui.pm:60]: I guess if our tests show that assets over CDN might not be always reachable then tools/generate-packed-assets should already apply the retry within itself.

Comment (ilausuch) [reply tests/install/openqa_webui.pm:60]: I didn't check if this script retries, but if not, we could move the functionality to tools/generate-packed-assets checking it

**Comment by ilausuch** [comment]
Following the @okurz review, I prepared this PR #4094

**Pull request closed without merging**